src/common/format.ts

```diff
@@ -86,3 +86,7 @@ export const getParcelOverviewString = (
         (collectionDatetime && ` @ ${dayjs(collectionDatetime).format("DD/MM/YYYY HH:mm")}`)
     );
 };
+
+export const formatTimeStringToHoursAndMinutes = (timeString: string): string => {
```

Comment:
This is to change 10:00:00 into 10:00

Reply:
Feels a bit dodgy to do it like that, especially if someone changes the format. Maybe convert to an actual date and then format the time? 
```
const time = new Date(timeString);
return `${time.getHours()}:${time.getMinutes()}`
```

Reply:
I think because timeString is HH:mm:ss it becomes invalid date when you put it in new Date(timeString).
I have now used dayjs instead but I need to pass in a 0 otherwise it becomes a single digit.

Reply:
Should we pad the hour as well?

```suggestion
    const hours = String(dayjsTime.hour()).padStart(2, "0");
    const minutes = String(dayjsTime.minute()).padStart(2, "0");
    return `${hours}:${minutes}`;
```

or

```
const padNumberWithLeadingZeros = (number: number, places: number) : String => {
    return String(number).padStart(places, "0");
}

export const formatTimeStringToHoursAndMinutes = (timeString: string): string => {
    dayjs.extend(customParseFormat);
    const dayjsTime = dayjs(timeString, "HH:mm:ss");
    return `${padNumberWithLeadingZeros(dayjsTime.hour())}:${padNumberWithLeadingZeros(dayjsTime.minute())}`;
};
```
mix and match with what you think is the nicest outcome